scripts/update/Updater.php

```diff
@@ -22,15 +22,30 @@
 namespace oat\taoOpenId\scripts\update;
 
 
+use oat\tao\model\accessControl\func\AccessRule;
+use oat\tao\model\accessControl\func\AclProxy;
 use oat\tao\scripts\update\OntologyUpdater;
+use oat\taoOpenId\model\RelyingPartyService;
 
 class Updater extends \common_ext_ExtensionUpdater
 {
     public function update($initialVersion)
     {
         if ($this->isVersion('0.0.1')) {
             OntologyUpdater::syncModels();
+            AclProxy::applyRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/generis.rdf#AnonymousRole', ['ext'=>'taoOpenId','mod' => 'Connect', 'act' => 'callback']));
```

Comment:
do not change previous update scripts, if something was missed in the past, please release a fix with a different version